api/conftest.py

```diff
@@ -63,6 +67,51 @@
 )
 from users.models import FFAdminUser, UserPermissionGroup
 
+
+def pytest_addoption(parser: pytest.Parser) -> None:
+    parser.addoption(
```

Comment:
Just confirming that this function will be auto-invoked without a decorator.

Reply:
See coverage report :)

Dive into [pytest hooks](https://docs.pytest.org/en/7.1.x/how-to/writing_hook_functions.html) if you're curious about how this works.